.sass-lint.yml

```diff
@@ -0,0 +1,220 @@
+linters:

```

Comment:
Any particular reason for `.sass-lint.yml` when we already have `.scss-lint.yml` ?